cmd/endpoint.go

```diff
@@ -142,6 +148,15 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 	}, nil
 }
 
+// isHostIPv4 - helper for validating if the provided arg is an ip address.
+func isHostIPv4(ipAddress string) bool {
+	host, _, err := net.SplitHostPort(ipAddress)
```

Comment:
I see given argument is in the form of IP:/path,  `ipAddress` as  argument is not correct.  The returned value of `net.SplitHostPort()` could be `IP` and `/path' (as port).  I think it would be more appropriate to swap names i.e. `ipAddress` becomes `host` vice versa.